app/src/main/java/com/keylesspalace/tusky/db/DraftsAlert.kt

```diff
@@ -0,0 +1,126 @@
+/* Copyright 2023 Andi McClure

```

Comment:
How do we do this? Do I just put my name here because I created the file?

Reply:
Best practice is `Copyright 2023 Tusky contributors` but I also don't mind if you put you own name